managed/resource_cluster.go

```diff
@@ -82,6 +82,24 @@ and modify the backup schedule of the cluster being created.`,
 						Type:     types.StringType,
 						Required: true,
 					},
+					"num_cores": {
```

Comment:
What happens if the customer sets num_cores and disk_size_gb  under `node_info` part of the TF and here also?

Reply:
If the root level `node_info` as well as the new per-region parameters `num_cores` and `disk_size_gb` are provided, then the created clusterSpec will contain the same fields (i.e. nodeInfo in root and per-region fields in clusterRegionInfoList). 

The apiserver logic will ensure that the per-region values take precedence over the root level value.